crates/lang/ir/src/ir/attrs.rs

```diff
@@ -723,13 +765,66 @@ impl From<InkAttribute> for Attribute {
     }
 }
 
+/// This function replaces occurrences of a `TokenTree::Ident` of the sequence
+/// `selector = _` with the sequence `selector = "_"`.
+///
+/// This is done because `syn::Attribute::parse_meta` does not support parsing a
+/// verbatim like `_`. For this we would need to switch to `syn::Attribute::parse_args`,
+/// which requires a more in-depth rewrite of our IR parsing.
+fn transform_wildcard_selector_to_string(grp: Group2) -> TokenTree2 {
```

Comment:
Yeah, no one is happy with this. There'll be a follow-up issue, feel free to implement it. I started but it quickly gets super-involved, since we need to change a lot of stuff for it.

Reply:
Yeah, no one is happy with this. There'll be a follow-up issue, it quickly gets super-involved because we can't mix `parse_meta` with the plain `syn::parse` API.